StRoot/StEvent/StFcsCollection.h

```diff
@@ -64,9 +67,9 @@ class StFcsCollection : public StObject {
     StSPtrVecFcsPoint   mPoints[kFcsNDet];  
 
     Int_t mFcsReconstructionFlag=0;     // undefined for now
```

Comment:
Just a suggestion, maybe put the initialization in the constructor would be better.